tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
+  "extends": "@tsconfig/node14/tsconfig.json",
   "compilerOptions": {
+    "lib": ["es2020"],
     "module": "es2020",
     "target": "es2020",
```

Comment:
We should keep in the mind, for web target should lower version. Best option is es5. That is supported by most of the browser.